### PR TITLE
busctl: add page

### PR DESCRIPTION
**Pull request by bobby-palmer**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Review by acuteenvy — COMMENTED** [review]
Thanks for your contribution. I have some suggestions and two more examples.

File: pages/linux/busctl.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# busctl
+
+> Introspect the bus.
```
Comment (acuteenvy) [inline pages/linux/busctl.md:3]:
```suggestion
> Introspect and monitor the D-Bus bus.
```

File: pages/linux/busctl.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+> More information: <https://www.freedesktop.org/software/systemd/man/busctl.html>.
+
+- Show all peers on the bus, by their service names:
+
+`busctl list`
+
+- Dump messages being exchanged:
+
+`busctl monitor`
```
Comment (acuteenvy) [inline pages/linux/busctl.md:12]:
```suggestion
- Dump messages being exchanged. If no service is specified, show all messages on the bus:

`busctl monitor {{service1 service2 ...}}`
```

File: pages/linux/busctl.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
+# busctl
+
+> Introspect the bus.
+> More information: <https://www.freedesktop.org/software/systemd/man/busctl.html>.
+
+- Show all peers on the bus, by their service names:
+
+`busctl list`
+
```
Comment (acuteenvy) [inline pages/linux/busctl.md:9]:
```suggestion
- Show process information and credentials of a bus service, a process, or the owner of the bus (if no parameter is specified):

`busctl status {{service|pid}}`

```

File: pages/linux/busctl.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+- Dump messages being exchanged:
+
+`busctl monitor`
+
```
Comment (acuteenvy) [inline pages/linux/busctl.md:13]:
```suggestion
- Show an object tree of one or more services (or all services if no service is specified):

`busctl tree {{service1 service2 ...}}`

```

**Comment by bobby-palmer** [comment]
@acuteenvy Thank you for the feedback!

**Review by acuteenvy — APPROVED** [review]
I'm sorry for the CI errors, I forgot to put newlines around the new examples.

**Review by kbdharun — APPROVED** [review]

**Pull request merged**